tests/main.cpp

```diff
@@ -28,12 +27,12 @@ QByteArray randomArray() {
 }
 
 bool testCrypto(QRSAEncryption::Rsa rsa) {
+
     QByteArray pub, priv;
     QRSAEncryption e;
 
-
     for (int i = 0; i < testSize; i++) {
-        e.generatePairKey(pub, priv, rsa);
+        QRSAEncryption::generatePairKeyS(pub, priv, rsa);
```

Comment:
if you want tests static methods create new test function for it 
but it not  necessary
necessary return "e.generatePairKey"